bucket/dfu-util.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.11",
+    "description": "DFU is intended to download and upload firmware to/from devices connected over USB",
+    "license": "GPL-2.0-only",
+    "homepage": "https://dfu-util.sourceforge.net/",
+    "url": "https://dfu-util.sourceforge.net/releases/dfu-util-0.11-binaries.tar.xz",
+    "hash": "6450de30a7dcd8d8c1273f43f0b153f054fd24d85f7f38296b1ad8edbd2ddb25",
+    "pre_install": [
+        "if ($architecture -eq 'arm64') { $architecture = '64bit' }",
```

Comment:
$architecture is an internal variable, so don't modify it. Just read it/check its value.